BLSQ/iaso
# POLIO-1102: scope only valid org units

In polio prod go to campaign CHD-2023-05-1_nOPV

In scope tab, the list will show both MICHEMIRE and MICHEMERE. The second one has been rejected. the name is actually an alias of the first one

Related JIRA tickets :POLIO-1102

## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes

only fetch valid org units

## How to test

Select a campaign, edit the scope and get the name of an org unit.
Edit the org unit and set the status to rejected.
Refresh the campaign scope, org unit should be present anymore

## Print screen / video


https://github.com/BLSQ/iaso/assets/12494624/bbefcff1-dd2f-4480-901e-e3c8a903c380
